data/json/recipes/weapon/mods.json

```
@@ -869,5 +869,18 @@
     "time": "45 m",
     "book_learn": [ [ "recipe_bows", 4 ], [ "textbook_weapwest", 5 ], [ "textbook_armschina", 5 ] ],
     "using": [ [ "blacksmithing_advanced", 5 ], [ "steel_standard", 1 ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "accelerator_gunmod",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "electronics",
+    "skills_required": [ [ "gun", 3 ] ],
+    "difficulty": 5,
+    "time": "20 m",
+    "book_learn": [ [ "manual_electronics", 5 ], [ "advanced_electronics", 4 ], [ "textbook_electronics", 5 ] ],
+    "using": [ [ "soldering_standard", 20 ] ],
```

Uses a decent chunk of wire already and the critical components all seem reasonably comparable to the light rail rifle but scaled down, so only thing I'd really say is vital to add is a couple tool qualities.
```suggestion
    "using": [ [ "soldering_standard", 20 ] ],
    "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
```
Adding a processor, ram, and some computer skill to represent the railgun mod needing to be a bit more complex to work with a potentially wide range of projectile velocities and sizes might be a reasonable optional addition too.